package.json

```diff
@@ -23,6 +23,8 @@
     "lodash-node": "^2.4.1",
     "object-fit-videos": "^1.0.3",
     "ophan-tracker-js": "1.3.9",
+    "preact": "^8.2.9",
+    "preact-compat": "^3.18.0",
```

Comment:
Don't forget to uninstall `react` and `react-dom` 🎉

Reply:
yeah v good point, cheers!

Reply:
all gone 👏